pkg/services/meta/meta.go

```diff
@@ -26,14 +28,18 @@ const (
 	rootKey = 0x00
 )
 
-// ContainerLister is a source of actual containers current node belongs to.
-type ContainerLister interface {
+// NeoFSNetwork describes current NeoFS storage network state.
+type NeoFSNetwork interface {
 	// List returns node's containers that support chain-based meta data and
 	// any error that does not allow listing.
 	List() (map[cid.ID]struct{}, error)
 	// IsMineWithMeta checks if the given CID has meta enabled and current
 	// node belongs to it.
 	IsMineWithMeta(cid.ID) (bool, error)
+	// Head returns actual object header from the NeoFS network (non-local
```

Comment:
objects are immutable so header cannot be other than actual

Reply:
"actual" means exist/lost. it hightly relates this issue: https://github.com/nspcc-dev/neofs-node/issues/3140